data/human/human missions.txt

```diff
@@ -4077,3 +4077,93 @@ mission "Laborer Payment"
+			label frigate
+			`	The man puts a hand on your shoulder and begins walking you toward the Frigate. As you approach, you note that the man's uniform doesn't have any Naval markings that you recognize.`
+			`	You enter the Frigate with the officer, and despite the decent crew requirement of a Frigate, you don't notice anyone else on board, and the interior of the ship appears more cushy than you would expect out of a military vessel. The officer leads you to what looks like a rather comfy lounge area of the ship. "Please, have a seat," he says.`
+			choice
+				`	(Take a seat.)`
+					goto seat
+				`	"No, I like to stand."`
+			`	"Suit yourself," the officer says with a shrug while taking a seat himself.`
+				goto questioning
+			label seat
+			`	You choose a table to sit at and the officer sits across from you.`
+			label questioning
+			`	"So what is it that you brought me here for?" you ask.`
+			`	The officer folds his hands together. "It has come to the Republic's attention that you've made contact with an alien species located to the north of the Republic. Is this correct?"`
+			choice
+				`	"It is. Is that a problem?"`
+					goto yes
+				`	"Perhaps. Why do you ask?"`
+					goto maybe
+				`	"Aliens? I don't know of any aliens up there."`
+			`	"Don't play coy with me, Captain. We are aware that you've made contact with the Hai.`
+				goto secret
+			label maybe
+			`	The officer chuckles. "I think the answer you're looking for is 'Yes,' not 'Perhaps.' We are aware that you've made contact with the Hai.`
+				goto secret
+			label yes
+			`	"There is no problem, Captain, but we are aware that you've made contact with the Hai.`
+			label secret
+			`	"But you may not be aware of the arrangement that the Republic has with the Hai. The Republic works hard to ensure that the existence of the Hai remains an open secret; we do not disallow captains such as yourself from stumbling upon them, but we do not allow this existence to be widely shared."`
+			choice
+				`	"Why are you telling me this?"`
+					goto why
+				`	"So you're threatening me to stay quiet?"`
+			`	"No threats, only a... warning, so to say.`
+				goto chaos
+			label why
+			`	"I'm telling you this as something of a... warning, so to say.`
+			label chaos
+			`	"I'm sure that you can imagine that should the existence of the Hai become public, it would cause a great deal of chaos, both for us and for the Hai, chaos that could easily end badly, particularly for the one who caused it. So it would be in everyone's interest if you kept this as our little secret."`
+			choice
+				`	"Alright. I won't let anyone know."`
+					goto ok
+				`	"And if I don't?"`
+			`	The officer leans back in his chair. "As I said, I'm giving you a warning that things could end badly for you, so you may want to think twice about revealing such information. Now, if you'll please depart my ship so that I can be on my way."`
```

Comment:
I feel like this line could serve to imply a little more. Right now, this is basically just the Republic issuing a veiled threat, even if it's presented as a warning. With a little tweaking though...
```suggestion
			`	The officer leans back in his chair. "As I said, I'm giving you a warning that things could end badly for you. We work hard to maintain it as an open secret but we also work hard to ensure the safety of our citizens. However, we are not the only ones who work hard to maintain the secret, so you may want to think twice about revealing such information. Now, if you'll please depart my ship so that I can be on my way."`
```

Reply:
We could do that, though that on its own to me feels like it has a connotation of there being a singular, other, more dangerous power working at it - whereas by at least calling out that they look after citizen safety, that implies that they are the biggest (or nearly so) of multiple powers working at it.

The first part can definitely be dropped though, I put him restating it just for emphasis but it's not necessary.